aesm-client/src/lib.rs

```diff
@@ -98,6 +98,36 @@ impl QuoteType {
     }
 }
 
+/*pub struct AesmKeyId {
+    key: Vec<u8>,
+}
+
+impl AesmKeyId {
+    pub fn key(&self) -> &[u8] {
+        &self.key
+    }
+}
+
+pub struct AesmKeyIds {
+    keys: Vec<u8>,
+}
+
+impl AesmKeyIds {
+    pub fn keys(&self) -> &[u8] {
+        &self.keys
+    }
+
+}
+*/
+#[derive(Debug)]
+pub struct QuoteInfoEx {
+    pub target_info: Vec<u8>,
+
+    pub pub_key_id: Vec<u8>,
+
+    pub pub_key_id_size: u64,
```

Comment:
This shouldn't be returned, it's `pub_key_id.len()`

Reply:
thanks, fixed